File: Assignment-34/Program1.java
import java.util.*;

class Program1
{
      public static void main(String arg[])
      {
            Scanner sobj = new Scanner(System.in);

            System.out.println("Enter The Size of Elements That You Want To Create:");
            int iSize = sobj.nextInt();

            int Arr[] = new int[iSize];
            
            Number obj = new Number();
            obj.Accept(Arr);
            
            System.out.println("Enter The Frequency Of elements");
            int iNo = sobj.nextInt();
          boolean bRet =  obj.Check(Arr,iNo);
          if(bRet==false)
          {
            System.out.println("TRUE");
          }
          else
          {
            System.out.println("FALSE");
          }
           
      }
}
class Number
{
      
      public void Accept(int Arr[])
      {
            Scanner sobj = new Scanner(System.in);
            System.out.println("Enter The Elements:");
            for(int icnt = 0;icnt<Arr.length;icnt++)
            {
                  Arr[icnt] = sobj.nextInt();
            }
      }
      public boolean Check(int Arr[],int iNo)
      {
            boolean bFlag = true;
            for(int i = 0;i<Arr.length;i++)
            {
                  if(Arr[i]==iNo)
                  {
                        bFlag = false;
                        break;
                  }
            }
            return bFlag;
      }

}

File: Assignment-7/program1.c
#include<stdio.h>

void Patter(int iNo)
{
      int iCnt = 0;

      if(iNo < 0)
      {
            iNo = -iNo;
      }

      for(iCnt = 1;iCnt <=iNo;iCnt++)
      {
            printf("$ * ");
      }
}
int main()
{
      int iValue = 0;

      printf("Enter The Number :");
      scanf("%d",&iValue);

      Patter(iValue);

      return 0;
}